app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/SearchResultsAdapter.java

```diff
@@ -53,7 +53,7 @@ public RssViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
     @Override
     public void onBindViewHolder(final RssViewHolder holder, int position) {
         Datum datum = datumList.get(position);
-        if (datum != null) {
```

Comment:
@chiragw15 we are getting datum out of datumList for the rss position so I it will not be null. It can be if there is some server error. So maybe we can have a check at top of datum and then datum.getTitle() inside it.